### DISCUSSION TITLE
No response to api calls sent from my site

### DISCUSSION THREAD

**Discussion by kabaevaa** [vimeo/player.js#1003] [Bug reports]
I'm developing the website https://acervomaraca.com.br/ and I'm dealing an issue with Vimeo. Many pages of this website have video galleries using videos from Vimeo. They worked fine until few weeks ago, but now they do not respond to api calls sent from my site. I'm using the DZS Video Gallery plugin and their support inform me that is a issue with Vimeo. I created some tokens to test and every single one didn't respond. You can verify the issue on this page https://acervomaraca.com.br/grupo/biu-roque for example.

Could you help me to solve this issue?

**Comment by bdougherty** [comment]
Without more details here, there's really nothing we can do.
